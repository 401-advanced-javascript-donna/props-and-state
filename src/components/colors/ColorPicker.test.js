import React from 'react';
import { shallow } from 'enzyme';
import ColorPicker from './ColorPicker';

describe('Color Display Component', () => {
  it('should render color', () => {
    const wrapper = shallow(<ColorPicker colors={
      [
        'red',
        'green',
        'black',
        'yellow'
      ]
    }/>);
    expect(wrapper).toMatchSnapshot();
  });
});
