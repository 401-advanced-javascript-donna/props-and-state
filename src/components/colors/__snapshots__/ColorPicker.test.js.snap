// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Display Component should render color 1`] = `
<Fragment>
  <button
    key="red"
    onClick={[Function]}
  >
    red
  </button>
  <button
    key="green"
    onClick={[Function]}
  >
    green
  </button>
  <button
    key="black"
    onClick={[Function]}
  >
    black
  </button>
  <button
    key="yellow"
    onClick={[Function]}
  >
    yellow
  </button>
</Fragment>
`;
